### PR TITLE
cookiecutter: modified files

### PR DESCRIPTION
**Pull request by cadenmyers13**

### CODE REVIEW

**Comment by cadenmyers13** [comment]
@sbillinge cookiecutter pt3

File: .github/workflows/docs.yml [hunk 1]
```diff
@@ -37,7 +37,7 @@ jobs:
         run: make -C doc html
 
       - name: Deploy
-        uses: peaceiris/actions-gh-pages@v4
+        uses: peaceiris/actions-gh-pages@v3
```
Comment (sbillinge) [inline .github/workflows/docs.yml:40]: not sure if we want to downgrade this.  Let's discuss.  In genera its better to go with a higher version unless there is a reason to downgrade.  Please ask to the whole group on slack. If the new version works we can update the cookiecutter.

**Review by sbillinge — COMMENTED** [review]
this looks way better, nice.  There is probably still a bit too much on this PR, making it difficult to merge, but this is definitely heading in the right direction.  I see a lot of thoughtfulness here and good decision making.

Please see inline comments.

**Comment by cadenmyers13** [comment]
Split into two PRs.

**Pull request closed without merging**